config/routes.rb

```diff
@@ -1,8 +1,13 @@
+require 'sidekiq/web'
```

Comment:
I think we should move this within the `if ENV['RAILS_ENV'] == 'development'` block since we only need it loaded for that env.